### ISSUE TITLE
Use the same sorting order as Everything

### ISSUE THREAD

**Issue by user1823** [srwi/EverythingToolbar#524]
### Preflight Checklist

- [X] I have searched all [open and closed issues](https://github.com/srwi/EverythingToolbar/issues?q=) for the same feature request without success.

### Problem Description

Often, the search term is a part of the name of several files. So, all of them show up in the search results. However, I am usually interested in the files that I have opened recently. The other files are essentially archived.

When I search in Everything, the files I have opened recently are at the top. But, in EverythingToolbar, they are sorted alphabetically. (I am talking about the default settings of both Everything and EverythingToolbar.) 

### Proposed Solution

EverythingToolbar should follow the same sorting order as Everything by default.

### Alternatives Considered

I tried changing the sort order in EverythingToolbar to Date Accessed but it asked me to enable Fast Sorting in Everything. When I tried to enable Fast Sorting in Everything, I noticed that it said "The search performance of Everything is reduced when sorting by date accessed."

I don't want to decrease the searching performance (I think it won't be a significant reduction but still...). So, I didn't enable that option.

I wonder why EverythingToolbar isn't using the same sorting order as Everything.

### Additional Information

Answering the question in https://github.com/srwi/EverythingToolbar/issues/467

> Are you sure it is actually sorted by date accessed in Everything? Looks like it is still sorting by name to me.

It seems to me that Everything sorts by name. BUT, if click on a search result to open it, it goes to the top and appears on the top on every subsequent search

Example:
In the following screenshot, note that Everything and EverythingToolbar display the files in the same order except for the one file at the top of Everything search results. This is the file that I opened recently by searching from Everything.
![image](https://github.com/user-attachments/assets/f0d5e2fb-63a7-4cda-9b00-789f30c4c3dd)

**Comment by srwi** [comment]
Hi @user1823, I think you have probably set your Everything to sort by "Date run", which I don't think is the default in Everything. It will sort all files that have been run from within Everything or EverythingToolbar by "Date run" first. All files that don't have a run date will fall back to sort by name after that. Here is a screenshot where I have set EverythingToolbar to the same sorting method as Everything:
![grafik](https://github.com/user-attachments/assets/ec7e4944-2ea3-4358-a815-a310918b50cc)

Automatically following the sorting method in Everything is currently not possible with the SDK, so unfortunately that can't be implemented.

**Comment by user1823** [comment]
> I think you have probably set your Everything to sort by "Date run"

I checked the settings and found that Everything is set to sort by Run Count (descending). I am sure that this is the default setting in Everything because I installed Everything yesterday only and I have not changed any settings.

> Automatically following the sorting method in Everything is currently not possible with the SDK

OK, but it would still make sense to use the same default sorting method as Everything. In other words, EverythingToolbar should use Run Count (descending) as the default sort order.
